en/docs/consume/invoke-apis/invoke-apis-using-tools/invoke-an-api-using-a-soap-client.md

```diff
@@ -11,24 +11,36 @@ Let's invoke the `PhoneVerification` API using a SOAP client.
 
     ![]({{base_path}}/assets/img/learn/subscribe-phone-verification.png)
 
-3.  Click the **Applications** menu, open the default application using which you subscribed to the API, and generate a production key.
-
-    ![]({{base_path}}/assets/img/learn/generate-keys-production-default.png)
-
-4.  Copy the access token to the clipboard as you need it later to invoke the API.
-
-5.  Download the SOAP UI installation that suits your operating system from <https://www.soapui.org/downloads/soapui.html> and open its console.
-6.  In the SOAP UI, right click on the **Projects** menu and create a new SOAP project.
+3.  Click the **Applications** menu, open the default application using which you subscribed to the API, and navigate to **OAuth2 Tokens** under **Production Keys**.
+    
+    ![]({{base_path}}/assets/img/learn/oauth2-phone-verification.png)
```

Comment:
As suggested above pls add ALT tags and make the image link zoomable

Reply:
Ack